src/backend/controllers/resources-controller.js

```diff
@@ -86,21 +87,32 @@ class ResourcesController extends BaseController {
     this.data.properties = this.data.currentResource.properties()
   }
 
+  // eslint-disable-next-line class-methods-use-this
+  flattenedFiltersProperties(query) {
+    return Object.keys(query)
+      .filter(key => key.includes('filters'))
+      .reduce((obj, key) => ({
+        ...obj,
+        [key]: query[key],
+      }), {})
+  }
+
   async findRecords({ query }) {
     this.data.perPage = 10
     const firstProperty = this.data.currentResource.decorate().getListProperties()[0]
-    const { page, sortBy, direction } = query
+    const { page, sortBy, direction, filters } = unflatten(query)
     this.data.page = Number(page) || 1
     this.data.sort = {
       sortBy: sortBy || firstProperty.name(),
       direction: direction || 'asc',
     }
-    const records = await this.data.currentResource.find({}, {
+    this.data.filters = this.flattenedFiltersProperties(query) || {}
+    const records = await this.data.currentResource.find(filters, {
```

Comment:
please update jsdoc for BaseResource#find with the filters. Remember base of your description other developers will build adapters so be sure to describe them well (for instance that `date` fields have `from` and `to`)